tests/contract_testcases/substrate/value/this_address.dot

```diff
@@ -3,7 +3,7 @@ strict digraph "tests/contract_testcases/substrate/value/this_address.sol" {
 	step1 [label="function step1\ncontract: b\ntests/contract_testcases/substrate/value/this_address.sol:3:13-62\nsignature step1()\nvisibility public\nmutability nonpayable"]
 	returns [label="returns\naddress payable "]
 	return [label="return\ntests/contract_testcases/substrate/value/this_address.sol:4:17-37"]
-	cast [label="cast address payable\ntests/contract_testcases/substrate/value/this_address.sol:4:24-37"]
+	cast [label="cast address payable\ntests/contract_testcases/substrate/value/this_address.sol:4:32-36"]
```

Comment:
The location of the cast is wrong now. If I compile this contract on your branch:
```
        contract b {
            function step1() public returns (address) {
                return int(this);
            }
        }
```
I get the following output:

```
$ cargo run tests/contract_testcases/substrate/value/this_address.sol --target substrate
   Compiling solang v0.1.10 (/home/sean/git/solang)
    Finished dev [unoptimized + debuginfo] target(s) in 12.82s
     Running `target/debug/solang tests/contract_testcases/substrate/value/this_address.sol --target substrate`
error: conversion from contract b to int256 not possible
  ┌─ /home/sean/git/solang/tests/contract_testcases/substrate/value/this_address.sol:4:28
  │
4 │                 return int(this);
  │                            ^^^^

error: not all contracts are valid
```
The location should be `int(this)`, not `this`. If I try the same on the main branch, I get:

```
$ cargo run tests/contract_testcases/substrate/value/this_address.sol --target substrate
   Compiling solang v0.1.10 (/home/sean/git/solang)
    Finished dev [unoptimized + debuginfo] target(s) in 28.16s
     Running `target/debug/solang tests/contract_testcases/substrate/value/this_address.sol --target substrate`
error: conversion from contract b to int256 not possible
  ┌─ /home/sean/git/solang/tests/contract_testcases/substrate/value/this_address.sol:4:24
  │
4 │                 return int(this);
  │                        ^^^^^^^^^